docs/primer.md

```diff
@@ -13,3 +15,314 @@
+
+$$C_{\text{FU}} = \left( 1 - v \right)P(1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t})\pi_{\text{FU}}\mu_{U}$$
+
+Now the 1 -- OR estimate of VE becomes:
+
+$$
+\begin{align*}
+VE &= 1 - \frac{C_{\text{FV}}C_{\text{NU}}}{C_{\text{FU}}C_{\text{NV}}} \\
+&= 1 - \frac{vP\lbrack\left( 1 - \varphi \right)(1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FU}}t})\pi_{\text{FU}} + \varphi(1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FV}}t})\pi_{\text{FV}}\rbrack\mu_{V})\left( \varepsilon_{\text{NU}}t\left( 1 - v \right)P\lambda_{\text{NU}}\pi_{\text{NU}}\mu_{U} \right)}{\left( \left( 1 - v \right)P(1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t})\pi_{\text{FU}}\mu_{U} \right)\left( \varepsilon_{\text{NV}}\text{tvP}\lambda_{\text{NV}}\pi_{\text{NV}}\mu_{V} \right)} \\
+&= 1 - \frac{\lbrack\left( 1 - \varphi \right)(1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FU}}t})\pi_{\text{FU}} + \varphi(1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FV}}t})\pi_{\text{FV}}\rbrack)\left( \varepsilon_{\text{NU}}\lambda_{\text{NU}}\pi_{\text{NU}} \right)}{\left( (1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t})\pi_{\text{FU}} \right)\left( \varepsilon_{\text{NV}}\lambda_{\text{NV}}\pi_{\text{NV}} \right)}
+\end{align*}
+$$
+
+As above, we will continue to assume that $\lambda_{\text{NV}} = \lambda_{\text{NU}}$ and $\pi_{\text{NV}} = \pi_{\text{NU}}$. And we will reimpose the assumption that $\varepsilon_{\text{FV}} = \varepsilon_{\text{FU}}$ and $\varepsilon_{\text{NV}} = \varepsilon_{\text{NU}}$, to better isolate the consequences of natural protection:
+
+$$= 1 - \left( 1 - \varphi \right) - \varphi\frac{1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FV}}t}}{1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t}}\frac{\pi_{\text{FV}}}{\pi_{\text{FU}}} = \varphi\left( 1 - \theta_{\pi}\frac{1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FV}}t}}{1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t}} \right)\text{}$$
+
+Because the term $\frac{1 - e^{- \varepsilon_{\text{FV}}\lambda_{\text{FV}}t}}{1 - e^{- \varepsilon_{\text{FU}}\lambda_{\text{FU}}t}}$ is not necessarily equal to θ~λ~, the TND VE estimate of 1 -- OR no longer necessarily reduces to the correct value of $\varphi(1 - \theta)$. In fact, 1 -- OR only reduces to the correct value of $\varphi(1 + \theta)$ under one of the following conditions:
+
+- $\lambda_{\text{FV}} = \lambda_{\text{FU}}$, that is, vaccination does not affect the probability of infection given exposure to the focal pathogen. In this case, $VE = \varphi\left( 1 - \theta_{\pi} \right) = \varphi\left( 1 - \theta \right)$ because $\theta = \theta_{\lambda}\theta_{\pi} = 1*\theta_{\pi} = \theta_{\pi}$. Note that vaccination may still affect the probability of ARI given infection, π. Thus, 1 -- OR can be unbiased under "leaky" vaccination, so long as vaccination reduces the probability of ARI, not infection itself.
```

Comment:
```suggestion
- $\lambda_{FV} = \lambda_{FU}$, that is, vaccination does not affect the probability of infection given exposure to the focal pathogen. In this case, $VE = \varphi\left( 1 - \theta_{\pi} \right) = \varphi\left( 1 - \theta \right)$ because $\theta = \theta_{\lambda}\theta_{\pi} = 1*\theta_{\pi} = \theta_{\pi}$. Note that vaccination may still affect the probability of ARI given infection, π. Thus, 1 - OR can be unbiased under "leaky" vaccination, so long as vaccination reduces the probability of ARI, not infection itself.
```